packages/contracts/src/utils.ts

```diff
@@ -258,23 +256,3 @@ const isValidStorageLayout = (storageLayout: any): boolean => {
 export const isNonNullObject = (obj: any): boolean => {
   return typeof obj === 'object' && obj !== null
 }
-
-export const getCurrentGitCommitHash = (): string | null => {
```

Comment:
I've moved this to the plugins package because this function is not used in contracts or core. It also was causing problems for me when I tried to import other functions (because it uses `execSync` which is not available in browsers).